package.json

```diff
@@ -16,14 +16,19 @@
   "author": "Einar Otto Stangvik <einaros@gmail.com> (http://2x.io)",
   "license": "MIT",
   "main": "index.js",
+  "exports": {
+    "import": "./wrapper.mjs",
+    "require": "./index.js"
+  },
   "browser": "browser.js",
   "engines": {
-    "node": ">=8.3.0"
+    "node": ">=10.0.0"
```

Comment:
I would recommend to make the jump to v12.

Reply:
v10 is officially no longer supported but only two months passed since its EOL date. I prefer to wait a little longer before dropping support for it, there is no additional maintenance burden compared to v12.